portugues/colabora/comdiferencas/diff/corrigeechamamae/corrigeechamamae.md

```diff
@@ -0,0 +1,3 @@
+Sua mãe entende de Git e diz "Não fez mais que sua obrigação. Aliás, nem o commit você fez...", e você:
+
+[Diz que é porque ainda não pensou em uma boa mensagem para o commit](naopensoumensagem/naopensoumensagem.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.